tests/test-coreos.sh

```diff
@@ -10,6 +10,14 @@ IMAGE_PATH=/tmp/rhcos-latest-image
 SCRIPT_FOLDER=$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P)
 RHCOS_SLB_REPO_PATH=${SCRIPT_FOLDER%/*}
 
+set_test_timeout() {
```

Comment:
What would be the cost of implementing this properly on the RHCOS CI?

Reply:
we could do that, it would make it easier for us, but I'm not sure it would be accepted there, as the situation is only because our current gitActions ci is slow... meaning it's very repo-specific, and doesn't really apply to their scenarios...
Plus, we might want to know that if from some reason their ci also fails on timeout (which is not our free gitActions servers)...

Reply:
update - got approval from them. will close this PR and do a much simpler one on cosa